src/convert.rs

```diff
@@ -170,6 +170,34 @@ where
     }
 }
 
+#[cfg(feature = "nalgebra")]
+impl<N, R, C, S> ToPyArray for nalgebra::Matrix<N, R, C, S>
+where
+    N: nalgebra::Scalar + Element,
+    R: nalgebra::Dim,
+    C: nalgebra::Dim,
+    S: nalgebra::Storage<N, R, C>,
+{
+    type Item = N;
+    type Dim = crate::Ix2;
+
+    fn to_pyarray<'py>(&self, py: Python<'py>) -> &'py PyArray<Self::Item, Self::Dim> {
+        unsafe {
+            let array = PyArray::<N, _>::new(py, (self.nrows(), self.ncols()), true);
```

Comment:
I just confirmed that it actually employs the Fortran layout! https://nalgebra.org/docs/faq/#what-is-the-memory-layout-of-matrices

Reply:
Added the link to the docs.